composer.json

```diff
@@ -29,15 +29,15 @@
         "paragonie/random_compat": "~1.1",
         "psy/psysh": "0.6.*",
         "swiftmailer/swiftmailer": "~5.1",
-        "symfony/console": "2.8.*|3.0.*",
-        "symfony/debug": "2.8.*|3.0.*",
-        "symfony/finder": "2.8.*|3.0.*",
-        "symfony/http-foundation": "2.8.*|3.0.*",
-        "symfony/http-kernel": "2.8.*|3.0.*",
-        "symfony/process": "2.8.*|3.0.*",
-        "symfony/routing": "2.8.*|3.0.*",
-        "symfony/translation": "2.8.*|3.0.*",
-        "symfony/var-dumper": "2.8.*|3.0.*",
+        "symfony/console": "3.0.*",
+        "symfony/debug": "3.0.*",
+        "symfony/finder": "3.0.*",
+        "symfony/http-foundation": "3.0.*",
+        "symfony/http-kernel": "3.0.*",
+        "symfony/process": "3.0.*",
+        "symfony/routing": "3.0.*",
+        "symfony/translation": "3.0.*",
+        "symfony/var-dumper": "3.0.*",
```

Comment:
> Because we now have a branch for 5.3. ;)

How does that even relevant with this changes? If 5.3 are meant for 3.1 then let's wait, otherwise there no reason to swap the deps to just increase commit count.